docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,74 +506,104 @@ If you submit a ledger API command (which can have multiple Daml commands), then
 
 Just putting all Daml commands into a single batch command does not help, as you might still create lots of views.
 
-In the protocol up to version 5, Canton creates a view for every action node in the transaction tree if the informees change 
-between the node and its parent. Therefore, in order to minimize the number of views, you should try to minimize the number of 
-changes of informees. Informees are roughly the signatories, observers of the contract and of the choice, and the controllers of the choice. 
+Strategy for Protocol Version 6
++++++++++++++++++++++++++++++++
 
-Instead of sending a batch with 20 exercise commands `Foo`, group your contracts by stakeholder and send one exercise 
-command on a generator contract with the list of contracts that in turn exercises the `Foo` choice per stakeholder group.
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the
+**participants that host their informees are not a subset of their parent view's informee participants**. Therefore, to
```

Comment:
Can you define the term "informee participant" first? AFAIK we have not defined this anywhere in our documentation. Since the term appears several times in this section, it might be worth to define it, say as, "the participants that must be informed about the view".